onboarding/src/Components/Employee/EmployeeMainPage.js

```diff
@@ -8,10 +8,10 @@ import EmployeeAccount from "./EmployeeAccount/EmployeeAccount.js";
 import EmployeeFormsList from "./EmployeeFormsList/EmployeeFormsList";
 import EmployeeForm from "./EmployeeForm";
 import QnAList from "./QnA/QnAList";
-
+import WelcomePage from "./WelcomePage";
 
 function EmployeeMainPage() {
-//    const [component, switchComponent] = useState(<EmployeeFormsList loggedUser={ loggedUser } switchPage={ loadForm } />);
+    const [welcomeView, setWelcomeView] = useState(true);
```

Comment:
We know there has to be `if` statement to check if user visits this page
first time.

Reply:
Of course, I'm waiting for proper api endpoint to finish this. For now it shows every time.